lms/djangoapps/discussion_api/tests/test_serializers.py

```diff
@@ -248,6 +251,12 @@ def test_following(self):
         serialized = self.serialize(self.make_cs_content({"id": thread_id}))
         self.assertEqual(serialized["following"], True)
 
+    def test_response_count(self):
+        thread_data = self.make_cs_content({})
```

Comment:
Pass an override value for the response count here rather than setting the default to 2.